views/address.tmpl

```diff
@@ -17,7 +17,7 @@
                     {{.Address}}<a
                         id="qrcode-init"
                         href="javascript:showAddressQRCode('{{.Address}}');"
-                        class="dcricon-qrcode jsonly no-underline color-inherit p10"
+                        class="dcricon-qrcode no-underline color-inherit p10"
```

Comment:
@gozart1 suggested having this CSS loaded into the global header by javascript in the initial page load:

```css
body .jsonly {
  display: block
}
```

Could you give that a shot (while retaining the `jsonly` class for this link)?

Reply:
ok. Noted.

Reply:
I'm not sure how @gozart1 would do this, but I thought he meant to inject the CSS

```css
.jsonly {
  display: block;
}
```

into the html header so it overrides the default in main.css of:

```css
.jsonly {
  display: none;
}
```

https://www.w3schools.com/cssref/pr_class_display.asp

Note that `block` does not mean to block it from displaying, it means to show it as a block element.

Reply:
So maybe in `<head>`:

```html
<script type="text/javascript">
  document.write("<style>body .jsonly {display :block }</style>");
</script>
```

Reply:
I made the page work by just adding `$(".jsonly").show()` to the page's css but I can still inject the  css.

Reply:
Its also been used here https://github.com/decred/dcrdata/blob/master/views/extras.tmpl#L99

Reply:
Gotcha.